.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "lib/forge-std"]
 	path = lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
-[submodule "lib/openzeppelin-contracts"]
-	path = lib/openzeppelin-contracts
-	url = https://github.com/openzeppelin/openzeppelin-contracts
+[submodule "lib/openzeppelin-contracts-upgradeable"]
```

Comment:
I think should be both modules:

`openzeppelin-contracts-upgradeable` and `openzeppelin-contracts`

 because 42 is not upgradable